Source/Dafny/DafnyOptions.cs

```diff
@@ -178,17 +211,21 @@ public enum IncludesModes { None, Immediate, Transitive }
               CompileTarget = CompilationTarget.Cpp;
             } else if (args[ps.i].Equals("php")) {
               CompileTarget = CompilationTarget.Php;
+            } else if (args[ps.i].Equals("python")) {
```

Comment:
Until this line, all the args were the same as the extension of these files. To ensure the least surprise, could you add also `py` as a recognized valid argument?

Reply:
thanks for input, went with it.